nemo/collections/asr/models/rnnt_bpe_models.py

```diff
@@ -179,7 +179,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             cfg.joint.num_classes = len(vocabulary)
             cfg.joint.vocabulary = ListConfig(list(vocabulary))
             cfg.joint.jointnet.encoder_hidden = cfg.model_defaults.enc_hidden
-            cfg.joint.jointnet.pred_hidden = cfg.model_defaults.pred_hidden
+            cfg.joint.jointnet.pred_hidden = cfg.decoder.prednet.get('proj_size', cfg.model_defaults.pred_hidden)
```

Comment:
Revert. We use model_defaults for exactly this reason. Inject values from model_defaults into other parts of the config as needed. The values of `pred_hidden` or `enc_hidden` or `joint_hidden` are top level values and must be used as is, common to all RNNT models.

```yaml

  model_defaults:
    enc_hidden: ${model.encoder.d_model}
    pred_hidden: 640
    joint_hidden: 640

```

Reply:
reverted.